typescript-loader/fixtures/app.ts

```diff
@@ -0,0 +1,4 @@
+import { sum, VERSION } from './utils.js';
```

Comment:
I feel like we shouldn’t include this feature, of “a specifier of `file.js` maps to `file.ts` that’s transpiled to JavaScript.” That feels beyond the scope of a TypeScript loader. I feel like a TypeScript loader should just transpile `.ts` specifiers and files on the fly, and that’s it.

Reply:
This is supposed to provide a real-world, workable, and above-all, useful solution. How would one facilitate type-checking as a lint step and also use `.ts`? `tsc --noEmit` will fail, causing a lint step in CI to fail; do you have another way to check types and also ignore that specific failure? (If so, huzzah and please do share 🙏)